src/main/java/io/reactivex/Observable.java

```diff
@@ -10277,9 +10277,8 @@ public final void safeSubscribe(Observer<? super T> s) {
     }
 
     /**
-     * Returns a Maybe that emits the single item emitted by this Observable if this Observable
-     * emits only a single item, otherwise if this Observable emits more than one item or no items, an
-     * {@code IllegalArgumentException} or {@code NoSuchElementException} is signalled respectively.
+     * Returns a Maybe that emits the single item emitted by this Observable or completes if this Observable is empty.
```

Comment:
The problem with two sentences is that most IDEs and the Javadoc method listing shows only the first sentence. Could you rewrite it that all relevant behavior is in the same sentence?